docs/reference/catalog-reference.md

```diff
@@ -1,6 +1,360 @@
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `search_parameters` | **dict / required**<br/>The catalog search parameters.                                                                                                              |
+| `as_dataframe`      | **bool**<br/>Determines how to return search results:</br/><ul><li>`True`: return GeoDataFrame.</li><li>`False`: return JSON.</li></ul>The default value is `True`. |
+
+<h5> Example </h5>
+
+```python
+catalog.search(
+    search_parameters=catalog.construct_search_parameters(
+        geometry=up42.get_example_aoi(),
+        collections=["phr"],
+        start_date="2022-06-01",
+        end_date="2022-12-31",
+        limit=20,
+        max_cloudcover=25,
+    ),
+    as_dataframe=False,
+)
+```
+
+### download_quicklooks()
+
+The `download_quicklooks()` function allows you to download low-resolution previews of scenes returned in search results.
+Visualize quicklooks with [`map_quicklooks()`](#map_quicklooks) or [`plot_quicklooks()`](#plot_quicklooks).
+
+```python
+download_quicklooks(
+    image_ids,
+    collection,
+    output_directory,
+)
+```
+
+The returned format is `list[str]`.
+
+<h5> Arguments </h5>
+
+| Argument           | Overview                                                                                              |
+| ------------------ | ----------------------------------------------------------------------------------------------------- |
+| `image_ids`        | **list[str] / required**<br/>The scene IDs.                                                           |
+| `collection`       | **str / required**<br/>The geospatial collection name.                                                |
+| `output_directory` | **Union[str, Path, none]**<br/>The file output directory. The default value is the current directory. |
+
+<h5> Example </h5>
+
+```python
+catalog.download_quicklooks(
+    image_ids=["a4c9e729-1b62-43be-82e4-4e02c31963dd"],
+    collection="phr",
+    output_directory="/Users/max.mustermann/Desktop/",
+)
+```
+
+## Orders
+
+### construct_order_parameters()
+
+The `construct_order_parameters()` function allows you to fill out an order form for a new catalog order.
+
+```python
+construct_order_parameters(
+    data_product_id,
+    image_id,
+    aoi,
+    tags,
+)
+```
+
+The returned format is `dict`.
+
+<h5> Arguments </h5>
+
+| Argument          | Overview                                                                                    |
+| ----------------- | ------------------------------------------------------------------------------------------- |
+| `data_product_id` | **str / required**<br/>The data product ID.                                                 |
+| `image_id`        | **str / required**<br/>The scene ID.                                                        |
```

Comment:
should it be "scene ID" or "image ID"?

Reply:
@seedlit Since the object being returned by `search()` was generally referred to as scenes I thought we should be consistent in using scene terminology. The object has `id` and `sceneId` but both are the same.

Reply:
seems a bit non-intuitive from user's perspective though.
Either we rename the argument to `scene_id` as well, or we update the description.
We need to be careful with renaming the argument because it will break existing user's pipelines.